src/processing/resources/page.js

```diff
@@ -14,6 +13,7 @@ const BODY_CREATED_EVENT_SCRIPT =
                var script = document.currentScript || document.scripts[document.scripts.length - 1];
                script.parentNode.removeChild(script);
           </script>`;
```

Comment:
Also use template on https://github.com/LavrovArtem/testcafe-hammerhead/blob/master/src/processing/resources/page.js#L86 and use template concatenation, to avoid new line in resulting string in https://github.com/LavrovArtem/testcafe-hammerhead/blob/master/src/processing/resources/page.js#L86 and https://github.com/LavrovArtem/testcafe-hammerhead/blob/master/src/processing/resources/page.js#L62